Error when running Svelte schematic in empty workspace: Cannot find module '@angular-devkit/core'

**Describe the bug**
When running `nx g @nxext/svelte:init` in an empty Nx Workspace, the command fails with ...

```
Cannot find module '@angular-devkit/core'
Require stack:
- C:\Users\Marvin\Projekte\playground-nx-svelte\node_modules\@nrwl\tao\src\commands\ngcli-adapter.js
- C:\Users\Marvin\Projekte\playground-nx-svelte\node_modules\@nrwl\tao\src\commands\generate.js
- C:\Users\Marvin\Projekte\playground-nx-svelte\node_modules\@nrwl\tao\index.js
- C:\Users\Marvin\Projekte\playground-nx-svelte\node_modules\@nrwl\cli\lib\init-local.js
- C:\Users\Marvin\Projekte\playground-nx-svelte\node_modules\@nrwl\cli\bin\nx.js
- C:\Users\Marvin\AppData\Roaming\npm\node_modules\@nrwl\cli\bin\nx.js
```

`isNxGenerator` returns false in `@nrwl/tao/src/commands/generate.js:111` which causes `ngcli-adapter` to be loaded.

```ts
if (ws.isNxGenerator(opts.collectionName, normalizedGeneratorName)) {
[...]
}
else {
    return (yield Promise.resolve().then(() => require('./ngcli-adapter'))).generate(cwd, Object.assign(Object.assign({}, opts), { generatorOptions: combinedOpts }), isVerbose);
}
```

`isNxGenerator` seems to be looking for a `cli` property in the schematic's schema...

```ts
isNxGenerator(collectionName, generatorName) {
    const schema = this.readGenerator(collectionName, generatorName).schema;
    return schema['cli'] === 'nx';
}
```

... which does not exist:

```js
{
  '$schema': 'http://json-schema.org/schema',
  '$id': 'Init',
  title: '',
  type: 'object',
  properties: {
    unitTestRunner: {
      description: 'Adds the specified unit test runner.',
      type: 'string',
      enum: [Array],
      default: 'jest'
    },
    e2eTestRunner: {
      description: 'Adds the specified e2e test runner.',
      type: 'string',
      enum: [Array],
      default: 'cypress'
    },
    skipFormat: {
      description: 'Skip formatting files.',
      type: 'boolean',
      default: false
    }
  },
  required: []
}
```

**To Reproduce**
```
 npx create-nx-workspace@latest nx-svelte

? What to create in the new workspace ...
> empty

cd nx-svelte
npm install @nxext/svelte --save
nx g @nxext/svelte:init
```

**Expected behavior**
From what I understand, it should not require Angular dependencies to work.

**Additional context**
```
  Node : 14.16.1
  OS   : win32 x64
  npm  : 7.24.0
  
  nx : 12.10.0
  @nrwl/cli : 12.10.0
  @nrwl/cypress : 12.10.0
  @nrwl/devkit : 12.10.0
  @nrwl/eslint-plugin-nx : 12.10.0
  @nrwl/express : 12.10.0
  @nrwl/jest : 12.10.0
  @nrwl/linter : 12.10.0
  @nrwl/node : 12.10.0
  @nrwl/tao : 12.10.0
  @nrwl/workspace : 12.10.0
  @nxext/svelte: 12.1.0
  typescript : 4.3.5
```

--- a/packages/svelte/src/executors/build/schema.json
+++ b/packages/svelte/src/executors/build/schema.json
@@ -3,6 +3,7 @@
   "title": "Build builder",
   "description": "",
   "type": "object",
+  "cli": "nx",
   "properties": {
     "project": {
       "type": "string",

--- a/packages/svelte/src/generators/application/schema.json
+++ b/packages/svelte/src/generators/application/schema.json
@@ -3,6 +3,7 @@
   "$id": "Svelte",
   "title": "",
   "type": "object",
+  "cli": "nx",
   "properties": {
     "name": {
       "type": "string",

--- a/packages/svelte/src/generators/component/schema.json
+++ b/packages/svelte/src/generators/component/schema.json
@@ -3,6 +3,7 @@
   "$id": "Component",
   "title": "",
   "type": "object",
+  "cli": "nx",
   "properties": {
     "name": {
       "type": "string",

--- a/packages/svelte/src/generators/init/schema.json
+++ b/packages/svelte/src/generators/init/schema.json
@@ -3,6 +3,7 @@
   "$id": "Init",
   "title": "",
   "type": "object",
+  "cli": "nx",
   "properties": {
     "unitTestRunner": {
       "description": "Adds the specified unit test runner.",

--- a/packages/svelte/src/generators/library/schema.json
+++ b/packages/svelte/src/generators/library/schema.json
@@ -3,6 +3,7 @@
   "$id": "Svelte",
   "title": "",
   "type": "object",
+  "cli": "nx",
   "properties": {
     "name": {
       "type": "string",

